data/example_dataset/json_tiling_data/tiled_output.json

```diff
@@ -1 +1 @@
"height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row1_col0", "timingDescription": "1 ms"}, {"scanCount": 1, "centerPointMicrons": {"x": 5144, "y": 6373}, "timingChoice": 7, "frameSizePixels": {"width": 2048, "height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row1_col1", "timingDescription": "1 ms"}, {"scanCount": 3, "centerPointMicrons": {"x": 14540, "y": -10830}, "fovSizeMicrons": 200, "timingChoice": 7, "frameSizePixels": {"width": 128, "height": 128}, "imagingPreset": {"preset": "Tuning", "aperture": "3", "displayName": "QC - 100\u00b5m", "defaults": {"timingChoice": 7}}, "standardTarget": "Moly Foil", "name": "MoQC", "notes": null, "timingDescription": "1 ms"}, {"scanCount": 1, "centerPointMicrons": {"x": 5142, "y": 6375}, "timingChoice": 7, "frameSizePixels": {"width": 2048, "height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row2_col0", "timingDescription": "1 ms"}, {"scanCount": 1, "centerPointMicrons": {"x": 5142, "y": 6371}, "timingChoice": 7, "frameSizePixels": {"width": 2048, "height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row0_col0", "timingDescription": "1 ms"}, {"scanCount": 3, "centerPointMicrons": {"x": 14540, "y": -10830}, "fovSizeMicrons": 200, "timingChoice": 7, "frameSizePixels": {"width": 128, "height": 128}, "imagingPreset": {"preset": "Tuning", "aperture": "3", "displayName": "QC - 100\u00b5m", "defaults": {"timingChoice": 7}}, "standardTarget": "Moly Foil", "name": "MoQC", "notes": null, "timingDescription": "1 ms"}, {"scanCount": 1, "centerPointMicrons": {"x": 5146, "y": 6375}, "timingChoice": 7, "frameSizePixels": {"width": 2048, "height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row2_col2", "timingDescription": "1 ms"}, {"scanCount": 1, "centerPointMicrons": {"x": 5146, "y": 6373}, "timingChoice": 7, "frameSizePixels": {"width": 2048, "height": 2048}, "imagingPreset": {"preset": "Normal", "aperture": "2", "displayName": "Fine", "defaults": {"timingChoice": 7}}, "sectionId": 8201, "slideId": 5931, "name": "row1_col2", "timingDescription": "1 ms"}, {"scanCount": 3, "centerPointMicrons": {"x": 14540, "y": -10830}, "fovSizeMicrons": 200, "timingChoice": 7, "frameSizePixels": {"width": 128, "height": 128}, "imagingPreset": {"preset": "Tuning", "aperture": "3", "displayName": "QC - 100\u00b5m", "defaults": {"timingChoice": 7}}, "standardTarget": "Moly Foil", "name": "MoQC", "notes": null, "timingDescription": "1 ms"}]}
```

Comment:
@ngreenwald just wanted them in here for demonstration purposes, I can remove them if it's too much clutter.

Reply:
These will get automatically generated when they run the script, right?

Reply:
Yes

Reply:
In that case I think it's fine to remove them. We have outputs in some of the other folders since they are used as inputs for subsequent functions, but that's not the case here.